requirements.txt

```diff
@@ -32,3 +32,4 @@ netifaces==0.10.4
 pyjokes==0.5.0
 psutil==4.1.0
 pep8==1.7.0
+-e git+https://github.com/ChristopherRogers1991/phue#egg=phue
```

Comment:
The upstream phue library does not have scene support. I added the functionality to make this skill. I've opened [this PR](https://github.com/studioimaginaire/phue/pull/64) to get the functionality into the upstream library, but until it's merged, we'll have to use mine.

Definitely a good call on using a specific commit. I've updated the requirements.